charts/remoting-agent/values.yaml

```diff
@@ -0,0 +1,239 @@
+# Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+---
+# Remoting Agent Deployment Values
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- Specifies how many replicas of a deployed pod shall be created during the deployment
+# Note: If horizontal pod autoscaling is enabled this setting has no effect
+replicaCount: 1
+
+image:
+  # -- Which derivate of the remoting agent to use
+  repository: ghcr.io/eclipse-tractusx/knowledge-agents/remoting-agent
```

Comment:
Do not use ghcr.io but Docker Hub. Please refer to [TRG 4.05 - Container registries](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05).

Reply:
moved to dockerhub/tractusx